tests/console/force_scheduled_tasks.pm

```diff
@@ -43,11 +43,11 @@ sub run {
     my $systemd_tasks_cmd = 'echo "Triggering systemd timed service $i" && systemctl start $i';
     $systemd_tasks_cmd .= ' && systemctl mask $i.{service,timer}' unless get_var('SOFTFAIL_BSC1063638');
     assert_script_run(
-'for i in $(systemctl list-units --type=timer --state=active --no-legend | sed -e \'s/\(\S\+\)\.timer\s.*/\1/\'); do ' . $systemd_tasks_cmd . '; done', 150);
+'for i in $(systemctl list-units --type=timer --state=active --no-legend | sed -e \'s/\(\S\+\)\.timer\s.*/\1/\'); do ' . $systemd_tasks_cmd . '; done', 300);
```

Comment:
please extract it to variable with sane name

Reply:
But there is no specific reason why the two variables need to share the same value

Reply:
If you like we can make that explicit by selecting different values though :)

Reply:
then add **two** variables with sane name :)

Reply:
I know you like wikis :) - https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
I am not sure if you are being serious now. Since when do you want to extract a variable for an assert_script_run timeout? A magic number is something different, like `if $foo == 42 launch_nuclear_missiles()`. Here it is different: The value is the number of seconds and of course it's rather arbitrary.